tests/PHPStan/Analyser/DynamicMethodThrowTypeExtensionTest.php

```diff
@@ -10,6 +10,12 @@ class DynamicMethodThrowTypeExtensionTest extends TypeInferenceTestCase
 	public function dataFileAsserts(): iterable
 	{
 		yield from $this->gatherAssertTypes(__DIR__ . '/data/dynamic-method-throw-type-extension.php');
+
+		if (PHP_VERSION_ID < 80000) {
+			return;
+		}
+
+		yield from $this->gatherAssertTypes(__DIR__ . '/data/dynamic-method-throw-type-extension-named-args.php');
```

Comment:
even though there is a non-php8 early return in the line before, this test-case seems to run on php 7.3/7.4